### ISSUE TITLE
Blocks videos on youtube and css on github

### ISSUE THREAD

**Issue by stefankeys** [Karmesinrot/Anifiltrs#28]
Enabling "Anifiltrs — 🍱 Anime streaming & Manga reading"​​​​​​​​​ or "Anifiltrs/Extra" blocks youtube videos and removes a lot of css from github.

**Comment by Karmesinrot** [comment]
Could you give sample urls and a copy of the logger. As there are no filters for github or youtube, they both work well on my end.

**Comment by Karmesinrot** [comment]
@stefankeys Are you using ublock origin legacy? If so that would explain the issues you've described. Yesterday I made commits to several filter to use line continuation, a feature supported in uBO starting v1.25.3b4 to ease maintainance of several csp filters(kissanime, 9anime, and gogoanime). Out of the three the [gogoanime](https://github.com/Karmesinrot/Anifiltrs/blob/e32eb795bcd8c5cef6b884e9f2c37c41907940b2/Anifiltrs%20Main/Anifiltrs%20main.txt#L195) is what's causing the issue as with other two they'll cause issues with their respective sites.

**Comment by Karmesinrot** [comment]
I've thought about this for quite some time and saw that many of the filters uses in the main list are not supported by legacy even prior to this issue. Reverting the changes made using the line continuation would be troublesome in mainting the said filters. I have also a habit of switching/testing new synthaxes and resources that arrive in the latests releases, thus I made a disclaimer for users using uBO legacy.

I'm sorry I'm afraid you'll have to not be able to use "Anifiltrs — 🍱 Anime streaming & Manga reading", but I've pushed a "Anifiltrs/Extra" which will allow you to use it.
